taskcluster/config.yml

```diff
@@ -18,46 +18,48 @@ taskgraph:
 # The list of valid stages that can be used with `target-stage and `start-stage`.
 # These get attached to tasks in `kinds`.
 valid-stages:
-    - clean-corpus
-    - clean-mono
+    - alignments-backtranslations
+    - alignments-distillation
+    - alignments-teacher
+    - all-pipeline
+    - all-pr-pipeline
+    - analyze-corpus
+    - analyze-mono
     - bicleaner
     - bicleaner-model
-    - merge-corpus
-    - merge-devset
-    - merge-mono
-    - train-vocab
-    - train-backwards
+    - clean-mono
+    - clean-parallel
+    - collect-backtranslations
+    - collect-distillation-mono
+    - collect-distillation-parallel
+    - distillation-parallel-keep-best
```

Comment:
This section definitely confuses me a lot. I'm thinking, what if we replace "parallel" with "all" for stages where we have merged the mono+parallel corpus? It wouldn't be confused with the original parallel corpus then. For example,  "distillation-all-keep-best." Would it be easier to understand?